torch/quantization/fake_quantize.py

```diff
@@ -65,9 +63,26 @@ def extra_repr(self):
             self.fake_quant_enabled, self.observer_enabled,
             self.scale, self.zero_point)
 
-default_fake_quant = FakeQuantize
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super(FakeQuantize, self)._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + 'quant_min'] = self.quant_min
```

Comment:
interesting, looks like we have a legit use cases for non-Tensor `register_buffer` (cc @gchanan). No action now, just interesting